docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -655,18 +655,20 @@ jobs:
-| platform  | string | Optional. Platform to use for the update. Can be `android`, `ios`, or `all`. Defaults to `all`.                                                                                                                                                                     |
-| branch    | string | Optional. Branch to use for the update. If not provided, the branch from the workflow run will be used. For manually run workflows you need to provide a value. Example: `${{ github.ref_name \|\| 'testing' }}`. Provide _either_ a branch or a channel, not both. |
-| channel   | string | Optional. Channel to use for the update. Provide _either_ a branch or a channel, not both.                                                                                                                                                                          |
+| Parameter        | Type   | Description                                                                                                                                                                                                                                                         |
+| ---------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| message          | string | Optional. Message to use for the update. If not provided, the commit message will be used.                                                                                                                                                                          |
+| platform         | string | Optional. Platform to use for the update. Can be `android`, `ios`, or `all`. Defaults to `all`.                                                                                                                                                                     |
+| branch           | string | Optional. Branch to use for the update. If not provided, the branch from the workflow run will be used. For manually run workflows you need to provide a value. Example: `${{ github.ref_name \|\| 'testing' }}`. Provide _either_ a branch or a channel, not both. |
+| channel          | string | Optional. Channel to use for the update. Provide _either_ a branch or a channel, not both.                                                                                                                                                                          |
+| private_key_path | string | Optional. Path to a private key file to use for code signing the update. This gets passed as `--private-key-path` to the EAS CLI.                                                                                                                                   |
```

Comment:
Do we maybe want to mention explicitly the setup where a user uploads a private key as EAS Environment Variable and references it as `$PRIVATE_KEY` here?

I tried to make a not of it here too : https://github.com/expo/universe/blob/2f143aa8d51821c7e1c306768787c46b18bdd544/server/www/src/data/entities/workflow/job/WorkflowUpdateJob.ts#L47